public/angular/templates/reports/show.html

```diff
@@ -1,47 +1,242 @@
-    </tr>
-    <tr ng-if="report.metadata.description">
-      <td class="detail-key" translate>Shared Content</th>
-      <td class="detail-value" ng-bind-html="report.metadata.description">{{ report.metadata.description }}</td>
-    </tr>
-  </tbody>
-</table>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col">
+
+    </div>
+    <div class="col-lg-9">
+      <div class="card">
+        <div class="card-header bg-gray">
+          <div class="btn-toolbar justify-content-between" role="toolbar" aria-label="Toolbar with edit confirmations" ng-show="currentUser.can('edit data')">
+            <div class="btn-group">
+              <button type="button" ng-click="$back()" class="btn btn-default mr-2">
+                <i class="fa fa-arrow-left" aria-hidden="true"></i>
+              </button>
+            </div>
+            <div class="right-side">
+              <div class="btn-group mr-4" role="group" aria-label="Flag/Unflag Buttons">
+                <button type="button" class="btn btn-info" ng-click="toggleFlagged(false)">
+                  <span class="mr-2" translate> Un-Flag </span>
+                  <i class="fa fa-flag-o" aria-hidden="true"></i>
+                </button>
+                <button type="button" class="btn btn-danger" ng-click="toggleFlagged(true)">
+                  <span class="mr-2" translate> Flag </span>
+                  <i class="fa fa-flag" aria-hidden="true"></i>
+                </button>
+              </div>
+              <div class="btn-group" role="group" aria-label="Incident Buttons">
+                <button type="button" ng-class="{highlight: someSelected()}" class="btn btn-light" ng-controller="IncidentSelectModalController" ng-click="(someSelected()? setIncident():null)" translate>
+                  Add to Incident
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="card-body">
+          <div class="table-responsive">
+            <table class="table table-hover table-bordered table-striped detail-table">
+              <tbody>
+              <!----------------- General Report Information ----------------->
+              <tr>
+                <th scope="row" class="detail-key" translate>Content</th>
+                <td colspan="10" class="detail-value" ng-bind-html="report.content">{{ report.content }}</td>
+              </tr>
+              <tr ng-if="report.metadata.description">
+                <th scope="row" class="detail-key" translate>Shared Content</th>
+                <td colspan="10" class="detail-value" ng-bind-html="report.metadata.description">{{ report.metadata.description }}</td>
+              </tr>
+              <tr>
+                <th scope="row" class="detail-key" translate>Author</th>
+                <td colspan="10" class="detail-value">
+                  <a ng-href="{{report.metadata.accountUrl}}" target="_blank">{{ report.author }}</a>
```

Comment:
This link looks clickable but **does nothing** on click for me:

<img width="365" alt="Screen Shot 2020-07-30 at 09 44 23" src="https://user-images.githubusercontent.com/2047062/88950371-68bec600-d249-11ea-825b-4b2c1d17da9a.png">

Reply:
Fixed in https://github.com/TID-Lab/aggie/commit/899a45fee7d0c0358cd6288564c7fa60b37ecfe3. Also added views for twitter posts.

Reply:
> * Incident view pages e.g. http://aggie.test/incidents/5f171dad1771033da8b7b174 seem to have wonky styles, they're the only page not consistent with everything else (the statsbar is rendered at the bottom and the spacing is weird)

Fixed with https://github.com/TID-Lab/aggie/commit/40d320741d82f098f25e9864f9bea898f2ad7f13.